Tests/Utils/XCTestCase+Extensions.swift

```diff
@@ -22,7 +23,11 @@ extension XCTestCase {
         self.waitForExpectations(timeout: self.defaultTimeout, handler: nil)
     }
 
-    func expectation(function: String = #function, name: String = "") -> XCTestExpectation {
+    func expectation(name: String, function: String = #function) -> XCTestExpectation {
         self.expectation(description: function + "-" + name)
```

Comment:
also, I'm realizing this will be `"someFunctionName-"` if no `name` is provided.

it would be nice to avoid having that trailing `-` if `name` is empty.

Reply:
Done.